### PR TITLE
README general cleanup

### LINKED ISSUE #43: README.md should be correct also for AnalyticsVideo
The same problem with a wrong API description also exists for `AnalyticsVideo`. It's necessary to update it in the same way as for `AnalyticsLive`.

image

### PR DESCRIPTION
**Pull request by lutangar**
I cleaned up the README and add a shorter example usage with modern syntax. 
Previous examples are now all presented in the sandbox as real world examples, and a link to this file has been added in the doc.

EDIT: Also  fix #43 by removing invalid period format mentions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "lib/"
   ],
   "author": "",
-  "license": "ISC",
+  "license": "Apache-2.0",
```
Comment (olivierapivideo) [inline package.json:23]: Why Apache2 ? I saw that in sme other projects we use MIT. I have no opinion on the matter, but we will have to harmonize all that one day

**Review by olivierapivideo — APPROVED** [review]
👍

File: package.json (hunk 1 above)
Comment (lutangar) [reply package.json:23]: The [LICENCE](https://github.com/apivideo/nodejs-sdk/blob/master/LICENSE) file is already an Apache2 licence.
But I can change that too.

Comment (olivierapivideo) [reply package.json:23]: Ah ok I didn't see, we are already gaining consistency so :) 
Nop its ok for me.

**Pull request merged**